esp/useful_scripts/server_setup/new_site.sh

```diff
@@ -365,28 +371,13 @@ then
     read THROWAWAY
 fi
 
-# Mailman setup
-# To reset: remove list manually
-if [[ "$MODE_MAILMAN" || "$MODE_ALL" ]]
-then
-    newlist -q $SITENAME-websupport $ADMINEMAIL $MAILMAN_LIST_PASSWORD
-    add_members -r - granite-websupport <<EOF
-$ADMINEMAIL
-EOF
-    echo "Created Mailman list $SITENAME-websupport with initial member $ADMINEMAIL."
-    
-    echo "Support list has been created.  Please check them by looking over the"
-    echo -n "output above, then press enter to continue or Ctrl-C to quit."
-    read THROWAWAY
-fi
-
 # Apache setup
 # To reset: remove appropriate section from Apache config
 if [[ "$MODE_APACHE" || "$MODE_ALL" ]]
 then
     cat >>$APACHE_CONF_FILE <<EOF
 #   $INSTITUTION $GROUPNAME (automatically generated)
-WSGIDaemonProcess $SITENAME processes=1 threads=1 maximum-requests=1000
+WSGIDaemonProcess $SITENAME processes=2 threads=1 maximum-requests=1000
```

Comment:
Ok, that sounds reasonable.  It may be that Apache's WSGI module just doesn't quite do what we want, unfortunately, but that sounds like a good thing to do for now.